aten/src/ATen/Context.h

```diff
@@ -111,6 +111,8 @@ class AT_API Context {
   void setBenchmarkCuDNN(bool);
   bool deterministicCuDNN() const;
   void setDeterministicCuDNN(bool);
+  bool cachePlanCuFFT() const;
+  void setCachePlanCuFFT(bool);
```

Comment:
Is there a reason to turn caching on/off on a global purpose, as opposed to just having caching on all the time or providing a caching/non-caching function?